src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -43,11 +90,20 @@ const getLastMenu = (handleSubmit, pristine, submitting) => {
   );
 };
 
+const AntiDuplicationField = (props) => (
```

Comment:
haha

Reply:
no idea how to rename, but sounds funny

Reply:
yeah, just tried to reduce duplication on sonarcloud. I don't think we need to do like this in a prod code though